data/functions.sql

```diff
@@ -1264,3 +1264,157 @@ BEGIN
   RETURN to_jsonb(coalesce(tags, ''::hstore) || coalesce(ne_tags, ''::hstore));
 END;
 $$ LANGUAGE plpgsql STABLE;
+
+-- Get names from Natural Earth if the wikidata tag matches
+CREATE OR REPLACE FUNCTION tz_get_ne_names(wikidata_id TEXT, place_tag TEXT)
+RETURNS JSONB AS $$
+DECLARE
+    ne_name_tags HSTORE DEFAULT NULL;
+BEGIN
+    IF wikidata_id IS NULL THEN
+        RETURN '{}'::jsonb;
+    END IF;
+
+  -- if it's a country, only look it up in the iso and tlc tables
+    IF place_tag='country' OR place_tag='unrecognized' THEN
+        SELECT hstore(ARRAY[
+            'ne_name_ar', i.name_ar,
```

Comment:
I'm curious about the choice to do array - wouldn't just a straight hstore be better?